{{cookiecutter.package_name}}/docs/requirements.txt

```
@@ -1,3 +1,4 @@
+-e .
```

```suggestion
-e .  # required to build API docs
```
Just to be more explicit about it